tdrs-backend/tdpservice/data_files/serializers.py

```diff
@@ -114,3 +114,9 @@ def validate_file(self, file):
         validate_file_extension(file.name)
         validate_file_infection(file, file.name, user)
         return file
+
+    def validate_section(self, section):
```

Comment:
Note: This is blocking FRA files to be created via API. If we need it right away, then we can definitely remove this

Reply:
i think this note is sufficient for me to update it as part of #3398

Reply:
confirmed. 
![3397P2](https://github.com/user-attachments/assets/5db10c46-6f80-435e-8365-7d451a7a3063)